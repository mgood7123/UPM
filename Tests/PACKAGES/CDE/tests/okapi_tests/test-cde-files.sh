#!/bin/sh

source okapi_test_common.sh
okapi_test_init # call init function

$OKAPI_BIN /CDE/strace-4.6/cde.c /home/pgbovine $TESTDIR
$OKAPI_BIN /CDE/scripts/coalesce/bsdiff-4.3.tar.gz /home/pgbovine $TESTDIR

pushd $TESTDIR > /dev/null
find . | xargs file | sort > contents.txt
popd > /dev/null

diff -u $TESTDIR/contents.txt $TESTNAME.golden
