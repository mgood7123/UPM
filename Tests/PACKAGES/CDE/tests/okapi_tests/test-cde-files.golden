./CDE:                                    directory
./CDE/scripts/coalesce/bsdiff-4.3.tar.gz: gzip compressed data, from Unix, last modified: Tue Aug 16 15:15:13 2005
./CDE/scripts/coalesce:                   directory
./CDE/scripts:                            directory
./CDE/strace-4.6/cde.c:                   ASCII C program text
./CDE/strace-4.6:                         directory
./contents.txt:                           empty
.:                                        directory
